rest_api/v2/components/index.yml

```
@@ -133,6 +133,57 @@ schemas:
         $ref: '#/schemas/CategoryName'
       id:
         $ref: '#/schemas/CategoryId'
+  CategorySqlId:
+    type: integer
+    description: Category id
+    example: 42
+  CategorySqlGmbId:
+    type: string
+    description: Google My Business Category Id ( unique )
+    example: 'gcid:climbing_gym'
+  CategorySqlName:
+    type: object
+    properties:
+      name:
+        type: string
+        description: Translated category name
+        example: salle d'escalade
+      country:
+        type: string
+        description: Country code
+        example: 'FR'
+  CategorySqlNames:
+    type: array
+    description: Category name translated for available countries
+    items:
+      $ref: '#/schemas/CategorySqlName'
+  CategorySqlLegacyIds:
+    type: array
+    description: old related partoo categories id
+    example: ['5109c3a237zdc544d8e267378', 'bf5179d3a237zdc576d8e2673']
+  CategorySqlState:
+    type: string
+    description: state of category ( OK or DEPRECATED )
+    example: 'OK'
+  CategorySqlBlockPushLong:
+    type: boolean
+    description:  Long description can't be set on GMB if a business own a category with this flag set to True
```

Maybe it is clearer written like this :)

```suggestion
    description:  Boolean indicating if the business long description can be pushed to GMB when the business belongs to this category
```

---

very more elegant :D